src/main/java/org/molgenis/vcf/report/model/AffectedStatus.java

```diff
@@ -0,0 +1,8 @@
+package org.molgenis.vcf.report.model;
+
+public enum AffectedStatus {
+  AFFECTED,
+  UNAFFECTED,
+  MISSING,
+  UNRECOGNIZED;
```

Comment:
UNRECOGNIZED should not be in this list? (https://phenopackets-schema.readthedocs.io/en/latest/pedigree.html#rstaffectedstatus)

Reply:
not sure, this is the content of the Enum as present in the Phenopackets lib

Reply:
ok, probably an error in the docs. lets leave it like this.